server/src-rsr/initialise.sql

```diff
@@ -179,3 +179,51 @@ LANGUAGE plpgsql AS $$
       END LOOP;
     END;
 $$;
+
+CREATE TABLE hdb_catalog.event_triggers
+(
+  id SERIAL UNIQUE,
+  name TEXT PRIMARY KEY,
```

Comment:
id should be primary key, name should have unique

Reply:
I am not using id anywhere, I can just remove it.